content/docs/api-reference/get_params.md

```diff
@@ -0,0 +1,90 @@
+# dvc.api.get_params()
+
+Returns all the <abbr>params</abbr> associated with the given
+<abbr>stage</abbr>.
```

Comment:
`dvc params diff` has a `--targets` argument which takes paths and not stages. Can we have some consistency between targets in `dvc params diff` and the first arg to this method since they both read params? I understand why it makes sense to use stages in your intended use case. Could both `dvc params diff` and `get_params()` accept either a stage or a path like some other dvc commands do? WDYT?

Reply:
Do you mean that we should extend `dvc params diff` to accept also accept a `stage` name as a target?

Reply:
Potentially, yes, although we could start by including paths in `get_params()`. Not sure if I like having an arg that could be either stages or paths, though. WDYT?

Reply:
Given your comment above, I think it's fine to leave this out. I was thinking it could be useful as a generic way to read multiple different different parameter file formats directly into a dict (even outside a dvc repo), but I don't think it's necessary.